src/Sentry/IMetricHub.cs

```diff
@@ -17,8 +17,14 @@ internal interface IMetricHub
     /// <summary>
     /// Starts a child span for the current transaction or, if there is no active transaction, starts a new transaction.
     /// </summary>
-    ISpan StartSpan(string operation, string description);
+    ISpan StartSpan(string name, string operation, string description);
 
     /// <inheritdoc cref="IHub.GetSpan"/>
     ISpan? GetSpan();
 }
+
+internal static class MetricHubExtensions
+{
+    public static ISpan StartSpan(this IMetricHub hub, string operation, string description) =>
+        hub.StartSpan(description, operation, description);
+}
```

Comment:
We could add an overload on StartSpan instead (to the `Hub` and to the `IMetricHub` interface). No more extension method that way. All of this is fluffing around because we have Transactions and Spans - if we didn't have those two different types of tracers, we'd only need one method (because the trace would either have a Name or it wouldn't).